model/src/main/java/org/cbioportal/genome_nexus/model/MutationAssessor.java

```diff
@@ -48,6 +51,15 @@ public void setInput(String input) {
         this.input = input;
     }
 
+    public String getHgvs() {
+        return hgvs;
```

Comment:
Previous code was overwriting the original `input` value returned by mutation assessor web service with the hgvs value. Now we are properly storing the hgvs value in a separate field.